dataframe-csv/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/csv.kt

```diff
@@ -12,17 +12,16 @@ import kotlin.reflect.typeOf
 
 public class CsvDeephaven(private val delimiter: Char = DelimParams.CSV_DELIMITER) : SupportedDataFrameFormat {
     override fun readDataFrame(stream: InputStream, header: List<String>): DataFrame<*> =
-        DataFrame.readCsv(inputStream = stream, header = header)
+        DataFrame.readCsv(inputStream = stream, header = header, delimiter = delimiter)
 
     override fun readDataFrame(file: File, header: List<String>): DataFrame<*> =
-        DataFrame.readCsv(file = file, header = header)
+        DataFrame.readCsv(file = file, header = header, delimiter = delimiter)
 
     override fun acceptsExtension(ext: String): Boolean = ext == "csv"
 
     override fun acceptsSample(sample: SupportedFormatSample): Boolean = true // Extension is enough
 
-    // if the user adds the dataframe-csv module, this will override old CSV reading method in DataFrame.read()
-    override val testOrder: Int = CSV().testOrder - 1
+    override val testOrder: Int = 20_000
```

Comment:
What does it mean?

Reply:
This is the test order for `guess.kt`, so the order in which `DataFrame.read()` tries the supported formats. CSV used to have 20,000, so now the new one gets the same order number.